### ISSUE TITLE
[RFC] Version Endpoint

### ISSUE THREAD

**Issue by Raistlfiren** [FAForever/api#115]
It would be nice if we could generate a badge or something that displays the current commit version of the API that is on the stable and test server in the README. That way we can quickly see what version is on each server since we are still in alpha stages on both.

**Comment by micheljung** [comment]
What do you mean by "a badge"? API itself versioning is a very controversal topc and I don't yet dare to have an opinion, but we could provide a route e.g. `/revision` that returns the currently deployed revision.

But our API is way beyond alpha.

**Comment by Raistlfiren** [comment]
Well, I was just hoping for some indication as to what revision the "stable" and "alpha" API are instead of having to ask. A badge is just what I came up with, so we could include it on the README and refer to it for anyone asking. Right now the only way to know is to test routes and see if it errors out with a 404 page. Unless we auto deployed to the stable and test server after a commit... Since the API is in alpha, it shouldn't really matter if an endpoint changes...

**Comment by micheljung** [comment]
Where would that README be?

**Comment by Raistlfiren** [comment]
The README for the API - https://github.com/FAForever/api/blob/develop/README.md

**Comment by micheljung** [comment]
Oh now I get it, you mean one of those dynamic github badges? Why didn't you just say so :P

**Comment by Raistlfiren** [comment]
Yeah that is what I meant. Sorry, I suck at explaining myself. lol. That is why I tried to say in my first comment "generate a badge or something". If this doesn't seem like a waste of time, I will try to get a PR for it sometime.
